### PR TITLE
Convert create and delete admin tests to cucumber

### PR DESCRIPTION
**Pull request by callumacrae**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -51,7 +51,7 @@ gem "tzinfo-data"
 gem "blueprinter"
 
 gem "govuk-components", ">= 1.1.3"
-gem "govuk_design_system_formbuilder", "~> 2.1", ">= 2.1.5"
+gem 'govuk_design_system_formbuilder', '~> 2.3.0b1'
```
Comment (kicferk1) [inline Gemfile]: What's this? A non-rubocop-compliant edge release? Did someone get their PR with data attributes merged today?

File: spec/cypress/support/step_definitions/common-interaction.js [hunk 2]
```diff
@@ -3,38 +3,62 @@ import { When, Then } from "cypress-cucumber-preprocessor/steps";
...
+
+When("I click on {string}", (element) => {
+  get(element).click();
+});
+
+When("I click on {string} containing {string}", (element, containing) => {
+  get(element).contains(containing).click();
 });
 
+
```
Comment (kicferk1) [inline spec/cypress/support/step_definitions/common-interaction.js:33]: You thought we would miss this cheeky double lime whitespace, but we didn't :D

File: spec/cypress/support/step_definitions/common-navigation.js [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
+Given("I am logged in as an {string}", (user) => cy.login(user));
+
+Given("scenario {string} has been ran", (scenario) => cy.appScenario(scenario));
 
 const pagePaths = {
   cookie: "/cookies",
   start: "/",
+  "admin admin": "/admin/administrators",
+  "admin admin creation": "/admin/administrators/new",
+  "admin admin confirm creation": "/admin/administrators/new/confirm",
```
Comment (kicferk1) [inline spec/cypress/support/step_definitions/common-navigation.js:13]: roger roger...

`admin index`, `admin creation`, `admin confirm creation`?

Comment (callumacrae) [reply spec/cypress/support/step_definitions/common-navigation.js:13]: yea i hate it too. the other alternative i was thinking about is "admin management" which assumes that admins are the only people who can create admins - which isn't an unfair assumption. it's super long though!

**Review by kicferk1 — APPROVED** [review]

**Pull request merged**